### DISCUSSION TITLE
There should be a way for a content creator to share their dislike count analytics and have a badge shown for it

### DISCUSSION THREAD

**Discussion by grrrbenji** [Anarios/return-youtube-dislike#744] [Ideas]
There would be a sub menu that allows for creators to turn on and off sharing data on any one video while still being able to set a general preferred default mode for all videos to be posted with. The badge should be simple and should appear near where the verified checkmark is located. The icon I propose is simply the RYD logo upside down and backwards so it is a thumbs up;
![favicon](https://user-images.githubusercontent.com/62861245/188242255-51806559-b19c-4fd0-ac3e-b7246a0eb610.jpg)
The implementation of this system could lead to overall better accuracy. The use of more accurate numbers and then using a system to show certification of contribution and accuracy of numbers would bring on more people into using it.

**Comment by sy-b** [comment]
Duplicate of https://github.com/Anarios/return-youtube-dislike/issues/130
